src/seedu/addressbook/data/person/address/Block.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person.address;
+
+/**
+ * Represents an Address's block field in the address book.
```

Comment:
No need to mention this belongs to address book. It could be any application _like_ address book.